SimpleInstaller/SimpleInstaller.fs

```diff
@@ -0,0 +1,86 @@
+namespace FSharp.Compiler.CodeDom
+
+// run InstallUtil.exe on this dll to register the provider in the GAC
+//
+// note: the provider build process does not sign the dll; you will need to
+// make a key with "sn -k" and add --keyfile:path_to_key to the OtherFlags
+// property in FSharp.Compiler.CodeDom.fsproj before building
+
+open System.ComponentModel
+open System.Configuration
+open System.IO
+open System.Runtime.InteropServices
+open System.Text
+open System.Xml
+
+[<RunInstaller(true)>]
+type SimpleInstaller() =
+    inherit Install.Installer()
+
+    static let machine_config_path =
+        let framework_path = RuntimeEnvironment.GetRuntimeDirectory()
+        framework_path + @"CONFIG\machine.config"
+    static let machine_config = new XmlDocument()
+    static do
+        machine_config.Load(machine_config_path)
+    static let codedom_node =
+        let cd =
+          machine_config.DocumentElement.SelectSingleNode("system.codedom")
+        if cd <> null then cd else
+        machine_config.CreateElement("system.codedom") :> XmlNode
+    static let compilers_node =
+        let compilers = codedom_node.SelectSingleNode("compilers")
+        if compilers <> null then compilers else
+        let elt = machine_config.CreateElement("compilers")
+        ignore <| codedom_node.AppendChild(elt)
+        elt :> XmlNode
+
+    static let file_extension = ".fs"
+    static let language = "f#;fs;fsharp"
```

Comment:
I suppose it doesn't hurt to add them, though I'm not sure the provider will produce a valid F# script file at the moment. I'll try it out now. If it doesn't work, we have two options: we can leave the extensions in (because a user could work around it by simply emitting snippets), or we can keep it as-is (because it's not as polished as could be).